.github/workflows/ci.yml

```
@@ -63,11 +83,23 @@ jobs:
         with:
           python-version: ${{ matrix.python }}
       - name: Pre-install
```

The names for each step should represent what is going on in this step. That way, if the step is skipped it makes sense.
```suggestion
      - name: Pre-install legacy
```